qa/zcash/smoke_tests.py

```diff
@@ -0,0 +1,800 @@
+#!/usr/bin/env python3
+#
+# Execute the standard smoke tests for Zcash releases.
+#
+
+import argparse
+import decimal
+import os
+import subprocess
+import sys
+import time
+
+from slickrpc import Proxy
```

Comment:
When running with python3 the script fails for me on this line because it can't import slickrpc. This works if I switch the script to use python2. I am not sure if this problem is unique to my setup.

Reply:
This might be that wonky issue I ran into while testing another stack. Using `pip3 install ...` might resolve this, but not sure. We can come back to look at this after Split.

Reply:
Yeah, this was just bad setup on my part...